website/src/blogs/install-wordpress-using-dynamic-nfs-provisioner.md

```diff
@@ -0,0 +1,217 @@
+
+        **Output**:
+        ```sh
+        [develop@develop ~]$ kubectl get po -n openebs
+        NAME                                           READY   STATUS    RESTARTS   AGE
+        openebs-nfs-provisioner-79b6ccd59-v8p6s        1/1     Running   0          4m12s
+
+        [develop@develop~]$ kubectl get sc
+        NAME                 PROVISIONER         RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+        openebs-kernel-nfs   openebs.io/nfsrwx   Delete          Immediate              false                  67s
+        ```
+
+     **Optional(installation of local-pv-provisioner)**
+        The below command will install OpenEBS local-pv provisioner along with NFS  provisioner, as stated above local-pv is used as backend StorageProvider in this blog
+
+        ```sh
+        helm install openebs openebs/openebs -n openebs --create-namespace \
+          --set legacy.enabled=false \
+          --set ndm.enabled=false \
+          --set ndmOperator.enabled=false \
+          --set localProvisioner.enabled=true \
+          --set localProvisioner.hostpathClass.name=openebs-hostpath \
+          --set localprovisioner.basePath=/var/openebs/hostpath \
+          --set nfs-provisioner.enabled=true  \
+          --set nfs-provisioner.nfsStorageClass.backendStorageClass=openebs-hostpath
+        ```
+
+        **Note**: `localprovisioner.basePath` defines the custom hostpath directory to provide storage for backend PVCs.
+
+### Install WordPress
+
+  [WordPress](https://en.wikipedia.org/wiki/WordPress) is one of the most famous blogging platforms and also it is bloggers first choice because of it's easy to handle nature. One can dynamically scale web servers of WordPress(content management systems [CMS]) by making use of Horizontal Pod Autoscaler feature of Kubernetes i.e increasing load will automatically scale the wordpress pods, users can install WordPress in multiple ways, but in this blog, we are going to install from its Helm repository by using [MariaDB](https://en.wikipedia.org/wiki/MariaDB) server to satisfy the requirements of WordPress server.
+
+  #### Add helm repo for WordPress
+
+  - Use the below command to add helm repo for WordPress
+    ```sh
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+    ```
+  - Once repo has been added successfully, update helm repo using the following command
+    ```sh
+    helm repo update
+    ```
+
+  #### Installing WordPress
+  Once the repo has been updated successfully, we can install WordPress referring to dynamic NFS provisioner storageclass. In this case storageclass will be openebs-kernel-nfs, if you have different storageclass then update the value of --set persistence.storageClass accordingly. WordPress also accepts various other parameters, for more information click [here](https://github.com/bitnami/charts/tree/master/bitnami/wordpress#parameters)
+
+  ```sh
```

Comment:
Do we need "sh" to be included in the block ?

Reply:
Actually, in Github MD mentioning with in "```sh" will look like below
```sh
echo "Running as script"
```
and if we omit "sh" then it will be something like below
```
echo "Without sh"
```

Reply:
![image](https://user-images.githubusercontent.com/51209534/140919808-26c9747e-a362-4fc1-a408-0ff915a5c59c.png)

The intention of the observation is that the "sh" is being visible. If that is intended, we can keep it as is.

Reply:
No, It should not visible

Reply:
It is due to some indentation problem, fixed it